src/applications/edu-benefits/complaint-tool/actions/schoolSearch.js

```diff
@@ -50,3 +41,24 @@ export function searchSchools({ institutionQuery, page }) {
     });
   };
 }
+
+export function selectInstitution({ city, facilityCode, name, state }) {
+  return {
+    type: INSTITUTION_SELECTED,
+    city,
+    facilityCode,
+    name,
+    state
+  };
+}
+
+export function setCannotFindSchool() {
```

Comment:
This is the only new action. The other actions were just alphabetized. 

by using set data, we can interact with the form state and have the custom component update things not directly under its control. The corresponding reducer for this action is in the forms library.